app/src/main/res/raw/sessions_ar.json

```diff
@@ -984,7 +984,7 @@
     },
     "language_id": "ja",
     "slide_url": "https://speakerdeck.com/player/45c463463fbe47518576240a53622cdc",
-    "movie_url": "https://www.youtube.com/watch?v=GMAXZPndBo0",
+    "movie_url": "https://www.youtube.com/watch?v=lmti0rNxbmI",
```

Comment:
I'm guessing when I added movie urls for the first time I added the wrong one, so here I modified it to add the right one.